src/main/webapp/cluster-nodes/cluster-nodes.html

```diff
@@ -72,10 +72,11 @@ <h2 class="card-pf-title">Add Node
             inetAddress:server.getInetAddress()})" title="View Node Details">
           <div class="card-pf card-pf-accented card-pf-aggregate-status card-pf-with-action">
             <h2 class="card-pf-title">
-              <span class="pficon" ng-class="server.isStopped()?'fa fa-power-off':'pficon-ok'"></span>{{server.name}}
+              <span class="pficon" ng-class="server.isStopped()?'fa fa-power-off':'pficon-ok'"></span>{{server.getHost()}}
```

Comment:
I think we should swap them: server name (which is unique across the cluster) in the card title and host name in the card body